source/gpu_perf_api_unit_tests/counter_generator_cl_tests.cc

```diff
@@ -179,6 +179,18 @@ TEST(CounterDllTests, OpenClCounterNamesByGeneration)
     VerifyCounterNames(kGpaApiOpencl, kGpaHwGenerationGfx11, FALSE, counterNames);
 }
 
+TEST(CounterDllTests, ClOpenCounterContext)
+{
+    VerifyHardwareNotSupported(kGpaApiOpencl, kGpaHwGenerationNvidia, FALSE);
+    VerifyHardwareNotSupported(kGpaApiOpencl, kGpaHwGenerationIntel, FALSE);
+
+    VerifyOpenCounterContext(kGpaApiOpencl, kGpaHwGenerationGfx8);
+    VerifyOpenCounterContext(kGpaApiOpencl, kGpaHwGenerationGfx9);
+    VerifyOpenCounterContext(kGpaApiOpencl, kGpaHwGenerationGfx10);
+    VerifyOpenCounterContext(kGpaApiOpencl, kGpaHwGenerationGfx103);
+    VerifyOpenCounterContext(kGpaApiOpencl, kGpaHwGenerationGfx11);
```

Comment:
Hm. I was just following what the other test methods above do. They only test 8,9,10,103 and 11. Am I ovelooking or misunderstanding something?

Reply:
I realize the potential confusion here - GPA no longer supports Gfx6 and Gfx7, but the enum values still exist. I'm guessing the tests for those two generations were removed rather than updated to VerifyHardwareNotSupported(..., *Gfx6|Gfx7).

Reply:
Done